README.md

```diff
@@ -1,45 +1,84 @@
 
-- **An Informational HIP** - describes an Hashgraph design issue, or provides general guidelines or information to the Hashgraph community, but does not propose a new feature. Informational HIPs do not necessarily represent Hashgraph community consensus or recommendation, so users and implementers are free to ignore Informational HIPs.
+## Purpose
 
-Each HIPs should only be one single key proposal and/or idea. The idea should be focused and only issue to one subject matter to be successful. If an issue only affects one client, it does not need a HIP. While changes that affect multiple clients, or defines a standard for multiple apps to use, it does require a HIP.
+The goal of HIPs is to have a place to propose new features, to collect community thoughts and input on a particular issue, and further to document all these subject matters in one place. It’s a great way to document these discussions and proposals [here on GitHub](https://github.com/hashgraph/HIPs), because any [revisions made on these text files will be recorded](https://github.com/hashgraph/HIPs/commits/master). 
 
-A HIP must meet certain minimum criteria. It must be clear and have a complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+## Types
 
-## HIP Work Flow
-Parties involved in this process are you, the HIP author, the HIP editors, and the Hashgraph Developers.
+There are two types of Hedera Improvement Proposals: 
 
-Evaluate your idea: Before you begin, ask the Hashgraph community first if your idea is original to save time. Make sure the idea is applicable to the entire community and not to one particular author. While it may be a good idea for an author it may not be the case for all areas of the Hashgraph community. An excellent place to discuss your proposal is here in the Issues section of this repository. You can start creating a more formalized language around your HIP.
+- A Standard HIP - describes any changes that affect most or all clients using or running Hedera Hashgraph, this can be a change to the network protocol, a transaction validity rules, proposed application standards/conventions, or any changes or addition that affects the interoperability of applications using Hedera Hashgraph. Standard HIPs can be broken down into the following categories: 
 
-Following is the process that a successful EIP will move along:
+   - Core: This type of proposal includes improvements to the core Hedera Hashgraph protocol and services (cryptocurrency, smart contracts, and file service). 
 
-[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+   - Networking: This type of proposal includes improvements around proposed improvements to network protocol specifications. 
```

Comment:
I believe this is carry over from a different improvement process Hashing Systems used as the baseline for the HIP doc. Can you think of a more clear delineation between proposals? Otherwise I'd agree we can just delete networking and lump into core.

Reply:
My instinct is to just delete it unless anyone can think of a reason to keep it.

Reply:
Nope, sounds good to me!

Reply:
Removed the networking item via 3a0d065 - feel free to resolve if you see fit